--- /dev/null	Thu Jan  1 01:00:00 1970
+++ chooser_lib.fd	Wed Feb  2 01:08:17 2022
@@ -0,0 +1,12 @@
+##base _ChooserBase
+##bias 30
+##public
+*--- functions in V40 or higher (Release 3.1) ---
+CHOOSER_GetClass()()
+AllocChooserNodeA(tags)(a0)
+FreeChooserNode(node)(a0)
+SetChooserNodeAttrsA(node,tags)(a0/a1)
+GetChooserNodeAttrsA(node,tags)(a0/a1)
+ShowChooser(o,w,xpos,ypos)(a0/a1,d0/d1)
+HideChooser(o,w)(a0/a1)
+##end
