--- dosextens.h.orig	Thu Feb  3 11:59:06 2022
+++ dosextens.h	Fri Mar 11 01:19:26 2022
@@ -248,7 +248,7 @@ struct DosLibrary {
     LONG    dl_A5;
     LONG    dl_A6;
     struct ErrorString *dl_Errors;	  /* PRIVATE pointer to array of error msgs */
-    struct timerequest *dl_TimeReq;	  /* PRIVATE pointer to timer request */
+    TimeRequest_Type   *dl_TimeReq;	  /* PRIVATE pointer to timer request */
     struct Library     *dl_UtilityBase;   /* PRIVATE ptr to utility library */
     struct Library     *dl_IntuitionBase; /* PRIVATE ptr to intuition library */
 };  /*	DosLibrary */
