--- gfxmacros.h.orig	Tue Apr 13 02:41:06 2021
+++ gfxmacros.h	Fri Feb  4 01:50:29 2022
@@ -54,9 +54,6 @@
 #define SafeSetOutlinePen(w,c)	  {if (GfxBase->LibNode.lib_Version<39) { (w)->AOlPen = c;(w)->Flags |= AREAOUTLINE;} else SetOutlinePen(w,c); }
 #define SafeSetWriteMask(w,m)	{if (GfxBase->LibNode.lib_Version<39) { (w)->Mask = (m);} else SetWriteMask(w,m); }
 
-/* synonym for GetOPen for consistency with SetOutlinePen */
-#define GetOutlinePen(rp) GetOPen(rp)
-
 #define BNDRYOFF(w)	{(w)->Flags &= ~AREAOUTLINE;}
 
 #define CINIT(c,n)	  UCopperListInit(c,n);
@@ -114,9 +111,6 @@
 					else \
 						SetWriteMask(w,m); \
 				} while (0)
-
-/* synonym for GetOPen for consistency with SetOutlinePen */
-#define GetOutlinePen(rp) GetOPen(rp)
 
 
 #define BNDRYOFF(w)	do { \
