--- prtbase.h.orig	Thu Feb  3 11:59:06 2022
+++ prtbase.h	Fri Mar 11 01:22:56 2022
@@ -150,7 +150,7 @@ struct   PrinterData {
 #define  pd_PIOR1 pd_ior1.pd_p1
 #define  pd_SIOR1 pd_ior1.pd_s1
 
-	struct timerequest pd_TIOR;	/* timer I/O request */
+	TimeRequest_Type pd_TIOR;	/* timer I/O request */
 	struct MsgPort pd_IORPort;	/* and message reply port */
 	struct Task pd_TC;		/* write task */
         /* PRIVATE: and stack space (OBSOLETE) */
