--- texteditor.h.orig	Thu Feb  3 11:59:08 2022
+++ texteditor.h	Fri Mar 11 01:33:14 2022
@@ -248,7 +248,7 @@ struct LeftBarMouseMessage
 	UWORD Qualifier; /* copied from InputEvent */
 	WORD X;		/* mouse position relative to upper left corner of the leftbar area for this line */
 	WORD Y;
-	struct timeval TimeStamp; /* copied from InputEvent */
+	TimeVal_Type TimeStamp; /* copied from InputEvent */
 };
 
 
